pkg/scheduler/DESIGN.md

```diff
@@ -248,35 +261,54 @@ This approach served two purposes:
 1. tenant fairness via a simple round-robin between all tenants with non-empty query request queues
 1. rudimentary tenant isolation via shuffle-shard assignment of noisy tenants to only a subset of queriers
 
-While this inter-tenant Quality-Of-Service approach has worked well,
-other QOS issues have arisen from the varying characteristics of Mimir's two "query components"
+While this inter-tenant Quality-of-Service approach has worked well,
+other QoS issues have arisen from the varying characteristics of Mimir's two "query components"
 utilized by the queriers to fetch TSDB data for executing queries: ingesters and store-gateways.
 
-### New Requirement: Queue Splitting by Query Component
+### New Requirement: Isolating Query Component Latency Effects
 
 Ingesters serve requests for recent data, and store-gateways serve requests for older data.
-While queries can overlap the time periods of data fetched by both query components,
+While queries can span the time periods served by both query components,
 many requests are served by only one of the two components.
 
 Ingesters and store-gateways tend to experience issues independently of each other,
 but when one component was in a degraded state, _all_ queries would wait in the queue behind the slow queries,
 causing high latency and timeouts for queries which could have been serviced by the non-degraded query component.
 
-### Phase 1: Query Component Selection by Round-Robin
+#### Goals
+
+The primary goal of the new design solution is to keep queries which only require the non-degraded query component
+moving through the queue and being processed, rather than waiting in the queue behind the slow queries.
+
+#### Non-Goals
+
+In real-world scenarios queries to the degraded query component are often slow enough to hit timeouts,
+and the majority of those queries will be expected to fail until the component recovers.
+
+#### Constraints
 
-In the first phase, we believed that it would be sufficient to duplicate the tenant queue splitting approach.
-We split the tenant queues further by query component, so that each tenant could have up to four queues.
+The solution should maintain high utilization of queriers while there are still any requests in the queue.
```

Comment:
I took this from something you added later in the benchmarks section

> secondary measure of success is the continued utilization of queriers while there are still any requests in the queue.

I figured it was better to put it at the top here as a constraint, since we do not have stuff in this particular doc about querier utilization from the benchmarks and simulations